en/docs/learn/ai-agent-integration.md

```diff
@@ -0,0 +1,91 @@
+# AI agent integration
+
+In this tutorial, you'll create an AI-powered math tutor assistant capable of handling a variety of mathematical queries. The agent will be equipped with tools to perform fundamental arithmetic operations and intelligently combine and execute these tools to address user questions. By the end of this tutorial, you'll have built an interactive math assistant that can help users solve problems and provide clear, step-by-step explanations.
+
+## Prerequisites
+
+### Obtain OpenAI API key
+
+- Sign up at [OpenAI](https://platform.openai.com/signup/).
+- Get an API key from the [API section](https://platform.openai.com/docs/api-reference/authentication).
+
+### Step 1: Create a new integration project
+
+1. Click on the **`Ballerina Integrator`** icon in the sidebar.
+2. Click on the **`Create New Integration`** button.
+3. Enter the project name as `MathTutor`.
+4. Select the project directory location by clicking on the **`Select Location`** button.
+5. Click the **`Create New Integration`** button to generate the integration project.
+   <a href="{{base_path}}/assets/img/ai-agent/ai-agent-step1.gif"><img src="{{base_path}}/assets/img/ai-agent/ai-agent-step1.gif" alt="Create a New Integration Project" width="70%"></a>
+
+### Step 2: Create an agent
+
+1. Click the `+` button on the Ballerina Integrator side panel or navigate back to the Design screen and click on **`Add Artifact`**.
+2. Select **`AI Chat Agent`** under the **`AI Agent`** artifacts.
+3. Provide a **`Name`** for the agent. It will take a moment to create an agent with the default configuration.
+4. After creating the agent, you can configure it with a model provider, memory, tools, roles, and instructions.
+   <a href="{{base_path}}/assets/img/ai-agent/ai-agent-step2.gif"><img src="{{base_path}}/assets/img/ai-agent/ai-agent-step2.gif" alt="Create an Agent" width="70%"></a>
+
+### Step 3: Configure the agent behavior
+
+1. Click on the **`AI Agent`** box to open the agent configuration settings.
+2. Define the agent's **`Role`** and provide **`Instructions`** in natural language. These instructions will guide the agent’s behavior and tasks.
+3. Click **`Save`** to finalize and complete the agent behavior configuration.
+   <a href="{{base_path}}/assets/img/ai-agent/ai-agent-step3.gif"><img src="{{base_path}}/assets/img/ai-agent/ai-agent-step3.gif" alt="Configure the Agent Behavior" width="70%"></a>
+
+### Step 4: Configure the agent model
+
+1. Locate the circle connected to the **`AI Agent`** box. This circle represents the LLM model used by the agent.
+2. Click on the circle to open the model configuration options.
+3. In the **`Select Model Provider`** dropdown, choose **`OpenAiProvider`**.
+4. Next, provide the OpenAI API key in the **`API Key`** input field.  
+   Note:> Since the API key is sensitive it's recommended to create a configurable named `openAiApiKey` and use it as the value for the **`API Key`** field.
```

Comment:
Shall we use Note block to add notes
https://squidfunk.github.io/mkdocs-material/reference/admonitions/#usage

Reply:
fixed [dccaec3](https://github.com/wso2/docs-bi/pull/59/commits/dccaec3b2f272498274c399ca09bad81ef84510e)